docs/cli/inspr.md

```diff
@@ -18,11 +18,11 @@ inspr [flags]
 
 ### SEE ALSO
 
-* [inspr apply](inspr_apply.md)	 - Applies changes to the connected cluster
+* [insprctl apply](inspr_apply.md)	 - Applies changes to the connected cluster
 * [inspr auth](inspr_auth.md)	 - 
```

Comment:
What's `inspr auth` ?

Reply:
since there is only documentation of it i assume its a planned command that later on was changed into something like inprov

Reply:
i opted to remove the files that have inspr_auth or just remove the mentioned of it in certain files. I assume that in case i made a mistake it will show itself in the next task of updating the documentation